### PR TITLE
Update spec file to work with aarch64 targets

### LINKED ISSUE #104: Add arm64 support?
Is there a chance to crosscompile contrac for arm64/xperia 10ii?
Thanks!

### PR DESCRIPTION
**Pull request by llewelld**

### CODE REVIEW

**Comment by llewelld** [comment]
Rebased.

**Review by bionade24 — APPROVED** [review]

File: rpm/harbour-contrac.spec [hunk 1]
```diff
@@ -83,10 +83,10 @@ fi
 %{_datadir}/icons/hicolor/*/apps/%{name}.png
 %{_datadir}/%{name}/translations
 %{_bindir}/contracd
-%{_libdir}/systemd/user/contracd.service
+%{_userunitdir}/contracd.service
```
Comment (bionade24) [inline rpm/harbour-contrac.spec:86]: Doesn't this also need a change here? https://github.com/llewelld/harbour-contrac/blob/25b36142001bb1880f6bd71f8c6734f1ca1f0836/contracd/contracd.pro#L92

**Review by bionade24 — CHANGES_REQUESTED** [review]

File: rpm/harbour-contrac.spec (hunk 1 above)
Comment (bionade24) [reply rpm/harbour-contrac.spec:86]: Nevermind it's fine, I overlooked the `/user/` and got confused.

**Review by bionade24 — APPROVED** [review]

**Comment by bionade24** [comment]
I'm doing merge commits for now. @llewelld please tell if you would like to switch to only rebase PRs on master.

**Pull request merged**